src/Console/Config.php

```diff
@@ -128,6 +128,10 @@ protected function doExecute()
 				throw new RuntimeException("$cat.$key is an array and can't be set using this command.");
 			}
 
+			if ($this->config->get($cat, $key) == $value) {
```

Comment:
This loose comparison is possibly contentious, but I'm not sure what variable type we can expect both from the Config adapter and the user prompt. I would say they both are strings, which would allow to use `===` instead.

Reply:
Agreed and fixed.  (Also, guess which developer spends most of his time writing C++ 😉)